front-end/src/components/RollCallComponents/ReturnButtonComponent.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+	<button
+		class="bg-blue-500 text-white py-2 px-4 rounded mt-2 mb-4 max-w-128 absolute top-2 left-16"
+		@click="goBack"
+	>
+		Retour en arrière
+	</button>
+</template>
+
+<script setup>
+const goBack = () => {
+	window.history.back();
```

Comment:
useRouter().go(-1) ?
We are using vue.js.

https://router.vuejs.org/api/interfaces/RouterHistory.html#Methods-go